Gemfile.lock

```diff
@@ -499,12 +505,6 @@ GEM
     nokogiri (1.12.5)
       mini_portile2 (~> 2.6.1)
       racc (~> 1.4)
-    nokogiri (1.12.5-arm64-darwin)
-      racc (~> 1.4)
-    nokogiri (1.12.5-x86_64-darwin)
-      racc (~> 1.4)
-    nokogiri (1.12.5-x86_64-linux)
-      racc (~> 1.4)
```

Comment:
Same comment as for `google-protobuf` above. These used to be necessary with Bundler and no longer appear to be. :100: